src/PrestaShopBundle/Entity/Repository/FeatureFlagRepository.php

```diff
@@ -83,6 +83,8 @@ public function enable(string $featureFlagName): void
         $featureFlag = $this->getByName($featureFlagName);
         if (null !== $featureFlag) {
             $featureFlag->enable();
```

Comment:
Why are we not using a command to enable feature? The domain logic is in a repository, it' strongly coupled ! I know it's not on the PR scope. I'am just wondering why?

Reply:
Yep indeed, Im pretty sure we could (and should?) have a cqrs command for this :thinking: